internal/shipwright/build/build_test.go

```diff
@@ -79,7 +79,7 @@ var _ = Describe("Build", Label("shipwright", "build"), func() {
 				Expect(metav1.IsControlledBy(object, owner)).To(BeTrue())
 			})
 			It("should have target namespace set to openshift build", func() {
-				Expect(object.Spec.TargetNamespace).To(Equal(common.OpenShiftBuildNamespaceName))
+				Expect(object.Spec.TargetNamespace).To(Equal(common.FetchCurrentNamespaceName()))
```

Comment:
I wouldn't invoke this function here - if this test runs within a Kubernetes container, the current namespace name could be _very_ different. In fact, if this runs within a Tekton task, the value here can change depending on how the task is configured.

Use `.To(Equal("openshift-builds"))` instead.